### ISSUE TITLE
Build issue on iOS

### ISSUE THREAD

**Issue by Alystrasz** [bmabir17/google_maps_flutter_heatmap#1]
Hello,
I would like to use heatmaps on my Flutter app, that's why I'm trying to use your fork.

They work well on Android, but not on iOS.
When building the app, Xcode throws `'Google-Maps-iOS-Utils/GMUHeatmapTileLayer.h' file not found` in `GoogleMapHeatmapController.h`.

When replacing the import with `#import "GMUHeatmapTileLayer.h"`, Xcode throws:
```text
Include of non-modular header inside framework module 'google_maps_flutter_heatmap.GoogleMapHeatmapController': '/Users/spirals/Library/Developer/Xcode/DerivedData/Runner-axcithruvnzhhsgmqorctsqkpxgk/Build/Products/Debug-iphonesimulator/Google-Maps-iOS-Utils/GoogleMapsUtils.framework/Headers/GMUHeatmapTileLayer.h'
```

Since I'm a total beginner in iOS development, I have no idea what's going on 😭

**Comment by bmabir17** [comment]
I am sorry that unfortunately, I don't have any iOS device to test the package on. :cry:

**Comment by jml0123** [comment]
Hi @bmabir17  any updates on this? `GMUHeatmapTileLayer.h` definitely exists. The actual path within ios folder is "../Pods/Google-Maps-iOS-Utils/src/Heatmap/GMUHeatmapTileLayer.h'. 

I even tried to import the Google-Maps-iOS-Utils in Podfile `pod 'Google-Maps-iOS-Utils', :git => "https://github.com/googlemaps/google-maps-ios-utils"`  but I am getting the file not found error

**Comment by jml0123** [comment]
Btw, you can emulate an iOS device in android studio via XCode's simulator.

**Comment by bmabir17** [comment]
Don't have a mac, have to buy one to run the simulator :(

**Comment by BhupinderRana** [comment]
i fixed ios build issue https://github.com/BhupinderRana/google_maps_flutter_heatmap but heat map not show in ios

**Comment by JacoNeto** [comment]
@BhupinderRana Why did you close your fork? It was very helpful in a project I was developing.

**Comment by BhupinderRana** [comment]
Hi

Actually it is not complete yet it has problem in ios.


Regards

Bhupinder Singh

Flutter Developer

Zimo infotech

[cid:4389455d-cdac-43ff-b4d7-51490f20d013]
________________________________

**Comment by aikenahac** [comment]
Any updates on this. @BhupinderRana the repository could not be found....

**Comment by catmaomao** [comment]
anyone resolve this issue? i have the same problem. i replace Google-Maps-iOS-Utils with GoogleMapsUtils then it can build,but nothing show on heatmap.i want to cry...

**Comment by aikenahac** [comment]
Haha, my product launch relies on this to work. I'll figure it out and let you know if/when I do.

**Comment by aikenahac** [comment]
@catmaomao [my fork](https://github.com/aikenahac/google_maps_flutter_heatmap) works on my project on iOS, so I suggest you check it out.

**Comment by aikenahac** [comment]
If you don't know how to put a Git repository as a package into pubspec, here you go.

```
google_maps_flutter_heatmap:
    git:
      url: https://github.com/aikenahac/google_maps_flutter_heatmap.git
      ref: master
```

Edit: You should properly format it when inserting, as  `.yaml` is sensitive to proper indentation.

**Comment by bmabir17** [comment]
@aikenahac So you applied the solution by @BhupinderRana . Does it resolves the build issue and show heatmap properly? 
If so I will merge the https://github.com/bmabir17/google_maps_flutter_heatmap/pull/4 and push a package update on pubspec.

**Comment by aikenahac** [comment]
@bmabir17 yes, I did that as I think his fork, or a similar was removed. So far it works, I will test it again today to be sure, and let you know in the next 2 hours.

**Comment by aikenahac** [comment]
Testing now

**Comment by aikenahac** [comment]
@bmabir17 finished testing, it fixes the build issues, but the heatmaps are not displayed...

**Comment by bmabir17** [comment]
@aikenahac oh, That's unfortunate. :cry:

**Comment by aikenahac** [comment]
Yeah, I'll try to further investigate this issue when I have the time.
